ui/app/src/components/Controls/DateTimePicker.tsx

```diff
@@ -157,12 +163,21 @@ const DateTimePicker = withStyles(dateTimePickerStyles)((props: DateTimePickerPr
     timePickerProps = {},
     timeZonePickerProps = {}
   } = props;
+  const classes = useStyles({});
   const [pickerState, setPickerState] = useState({
     dateValid: true,
     timeValid: true,
     timezoneValid: true
   });
 
+  const autoCompleteEl = useRef(null);
+
+  useEffect(() => {
+    const autoCompleteButton = autoCompleteEl.current.querySelector('.MuiAutocomplete-popupIndicator');
+    autoCompleteButton.tabIndex = 0;
+  }, [autoCompleteEl]);
```

Comment:
Didn't realise this was a problem of the control/lib. Let's leave as it was then (rollback).

Reply:
done